src/pages/Selected/components/UserInfo.jsx

```diff
@@ -1,43 +1,231 @@
+        'Content-Type': 'application/json;charset=utf-8',
+        Authorization: localStorage.getItem('token'),
+      },
+      body: JSON.stringify({
+        orderName: userInfo.info.testName,
+        totalAmount: 3000,
+        testId: seatId,
+        seatInfo: testSeats,
+      }),
+    })
+      .then(res => res.json())
+      .then(data => {
+        loadTossPayments(clientKey).then(tossPayments => {
+          tossPayments.requestPayment('카드', data).catch(error => {
+            if (error.code === 'USER_CANCEL') {
+              alert('결제창이 닫혔습니다.');
+            } else if (error.code === 'INVAILD_CARD_COMPANY') {
+              alert('유효하지않은 카드입니다');
+            }
+          });
+        });
+      });
+  };
+
+  const isPhoneVaild = phone.length >= 9;
+  const codeUnderSix = phoneCheck.length < 6;
+  const codeOverTen = phoneCheck.length > 10;
+  const isEmailVaild = email.includes('@');
+  const isAllChecked =
+    testSeats !== '' &&
+    name !== '' &&
+    isPhoneVaild &&
+    phoneCheck !== '' &&
+    isEmailVaild;
+
+  const makeWarning = id => {
+    if (id === 'phone') {
+      return !isPhoneVaild ? '9자 이상 가능합니다.(숫자만)' : '가능합니다!';
+    }
+    if (id === 'phoneCheck') {
+      return (
+        (codeUnderSix && '6자리를 입력해주세요.') ||
+        (codeOverTen && '10자리까지 가능합니다.')
+      );
+    }
+    if (id === 'email') {
+      return !isEmailVaild && '이메일 주소를 다시 확인해주세요.';
+    }
+  };
 
-export default function UserInfo() {
   return (
     <>
-      {INPUT_LIST.map(({ id, title, placeholder }) => (
-        <>
-          <S.UserInfoContainer key={id}>
+      {INPUT_LIST.map(({ id, title, placeholder, type, handleKeyDown }) => (
+        <React.Fragment key={id}>
+          <S.UserInfoContainer>
             <S.Labels htmlFor={id}>{title}</S.Labels>
             <S.Inputs
               id={id}
-              disabled={id.includes('exam')}
+              disabled={id.includes('test')}
               placeholder={placeholder}
+              name={id}
+              value={userInfo.info[id]}
+              onChange={userInfo.handleInput}
+              type={type}
+              onKeyDown={handleKeyDown}
             />
-            {id.includes('phone') && (
-              <S.SmallBtn color="primary">
-                {id === 'phone' ? '인증' : '확인'}
+            {id === 'phone' && (
+              <S.SmallBtn color="primary" onClick={e => handlePhoneCheckBtn(e)}>
+                인증
+              </S.SmallBtn>
+            )}
+            {id === 'phoneCheck' && (
+              <S.SmallBtn color="primary" onClick={e => handleSubmitCode(e)}>
+                확인
               </S.SmallBtn>
             )}
           </S.UserInfoContainer>
-          {id === 'phone' && (
-            <S.WarnningMsg>9자 이상 가능합니다.(숫자만)</S.WarnningMsg>
-          )}
-          {id === 'email' && (
-            <S.WarnningMsg>이메일 주소를 다시 확인해주세요.</S.WarnningMsg>
-          )}
-        </>
+          <S.WarnningMsg>{makeWarning(id)}</S.WarnningMsg>
```

Comment:
```suggestion
          <S.WarnningMsg>{inputMessageCondition[id]}</S.WarnningMsg>
```

위의 리뷰를 토대로 이렇게 구현할 수 있겠네요!